corehq/apps/reports/templates/reports/filters/bootstrap5/single_option.html

```diff
@@ -2,22 +2,32 @@
 {% load hq_shared_tags %}
 {% block filter_content %}
   {% if pagination.enabled %}
-    <select id="{{ css_id }}"
-            class="input form-control report-filter-single-option-paginated"
-            data-handler="{{ pagination.handler }}"
-            data-url="{{ pagination.url }}"
-            data-action="{{ pagination.action }}"
-            style="width: 100%"  {# todo B5: inline style #}
-            data-placeholder="{{ select.default_text }}"
-            data-selected="{{ select.selected|default:'' }}"
-            name="{{ slug }}"></select>
+    <select
+      id="{{ css_id }}"
+      class="input form-control report-filter-single-option-paginated w-100"
```

Comment:
```suggestion
      class="input form-select report-filter-single-option-paginated w-100"
```

Reply:
thanks. likely wasn't seen because of the select2 styling that eventually dominates the select
fixed in https://github.com/dimagi/commcare-hq/pull/35704/commits/6fd62f71fcbd2fc56936b57f3388471d4498df88